src/app/presentation/introjs-directive/introjs.directive.ts

```diff
@@ -8,12 +8,16 @@ import {introJs} from 'intro.js/intro.js';
 export class IntrojsDirective implements OnInit {
 
   constructor(public slide: SlideComponent) {
+    if (!localStorage.numTours) {
+      localStorage.numTours = 0;
+    }
   }
 
   ngOnInit() {
     this.slide.onActive.subscribe((active) => {
-      if (active) {
+      if (active && localStorage.numTours <= 2) {
```

Comment:
There are two different tours; not an elegant solution to be sure. Definitely open to improvement ideas 🤓